packages/@o3r/localization/package.json

```diff
@@ -38,8 +40,6 @@
     }
   },
   "dependencies": {
-    "@o3r/dev-tools": "workspace:^",
-    "@o3r/schematics": "workspace:^",
```

Comment:
sure it's not needed to run ng add ?
@mrednic-1A could you clarify please ?

Reply:
it is needed for ng add indeed.
But we adopted a strategy to be able to keep the `@o3r/schematics` as peer dep.
Actually we run the `ng add` and if it's failing when trying to use `o3r/schematics` we display a custom message to the user saying that he needs to install schematics first. 

There is a second usecase when the package has `o3r/core` as peer dependency. Then we will display in the custom message that the user needs to install `o3r/core` directly because this one will install `o3r/schematics` because it has it as a direct dependency. 

Here is the example of the ng add for localization which uses that technique.
packages/@o3r/localization/schematics/ng-add/index.ts